spec/dummy_app/db/migrate/20110208155312_set_up_test_tables.rb

```diff
@@ -36,17 +36,18 @@ def up
       t.string :name, null: false
     end
 
-    # Classes: Vehicle, Car, Truck
+    # Classes: Vehicle, Car, Bicycle, Truck
     create_table :vehicles, force: true do |t|
       t.string :name, null: false
       t.string :type, null: false
-      t.timestamps null: false
+      t.integer :owner_id
+      t.timestamps null: false, limit: 6
     end
 
     create_table :skippers, force: true do |t|
       t.string     :name
-      t.datetime   :another_timestamp
-      t.timestamps null: true
+      t.datetime   :another_timestamp, limit: 6
+      t.timestamps null: true, limit: 6
```

Comment:
Ugh. MySQL's lack of Fractional Seconds Precision (FSP) has been a big headache on this project. I'm OK with this change, but can you remind me what datatype this produces in MySQL? Is it a `DATETIME(6)`? Does that mean six decimal points of FSP?

I wonder if, after this change, we can use less `timecop` in our tests .. but that's a story for another time. :smile: